src/Features/Core/Portable/FindUsages/AbstractFindUsagesService_FindImplementations.cs

```diff
@@ -183,31 +193,21 @@ private static async Task<ImmutableArray<ISymbol>> FindSourceAndMetadataImplemen
                 }
             }
 
-            if (!symbol.IsInterfaceType() &&
-                !symbol.IsAbstract)
-            {
-                implementationsAndOverrides.Add(symbol);
-            }
```

Comment:
primary change was that instead of unilaterally adding hte original symbol back in, we only do it if we find no actual overrides/impls.